apps/scan/src/main/openapi/scan.yaml

```diff
@@ -1551,6 +1551,63 @@ paths:
         "404":
           $ref: "../../../../common/src/main/openapi/common-external.yaml#/components/responses/404"
 
+  /v0/events/:
+    post:
+      x-jvm-package: scan
+      operationId: "getEventHistory"
+      description: |
+        Returns the event history in ascending order, paged, from ledger begin or optionally starting after a record time.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/EventHistoryRequest"
+      responses:
+        "200":
+          description: ok
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/EventHistoryResponse"
+        "400":
+          $ref: "../../../../common/src/main/openapi/common-external.yaml#/components/responses/400"
+        "500":
+          $ref: "../../../../common/src/main/openapi/common-external.yaml#/components/responses/500"
+  /v0/events/{event_id}:
+    get:
+      x-jvm-package: scan
+      operationId: "getEventById"
+      description: |
+        Returns the event with the given update_id.
+      parameters:
+        - name: "event_id"
+          in: "path"
+          required: true
+          schema:
+            type: string
+        - name: "lossless"
```

Comment:
This should not be used (was part of a deprecated version of updates, /v0/updates

Reply:
Yes, fixed